Libraries/ART/ReactNativeART.js

```diff
@@ -205,9 +211,15 @@ function extractOpacity(props) {
 // ReactART.
 
 var Group = React.createClass({
-
+  contextTypes: {
+    isInSurface: React.PropTypes.bool.isRequired,
+  },
   render: function() {
     var props = this.props;
+    invariant(
+      this.context.isInSurface,
+      'ART: <Group /> must be a child of a <Surface />'
```

Comment:
Is there enough context to easily find what component caused this error? Maybe invariant stack trace is enough.